ultra/tests/test_tune.py

```diff
@@ -0,0 +1,112 @@
+# MIT License
+#
+# Copyright (C) 2021. Huawei Technologies Co., Ltd. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+import glob
+import os
+import shutil
+import unittest
+
+from ultra.tune import _AVAILABLE_TUNE_METRICS
+from ultra.utils.episode import LogInfo
+
+
+class TuneTest(unittest.TestCase):
+    # Put generated files and folders in this directory.
+    OUTPUT_DIRECTORY = "tests/tune_test/"
+
+    def test_tune_cli(self):
+        TASK = "00"
+        LEVEL = "easy"
+        POLICY = "ppo"
+        EPISODES = 2
+        MAX_EPISODE_STEPS = 50
+        TIMESTEP = 0.1
+        HEADLESS = True
+        EVAL_EPISODES = 2
+        SAVE_RATE = 1
+        SEED = 2
+        LOG_DIR = os.path.join(TuneTest.OUTPUT_DIRECTORY, "tune_logs/")
+        CONFIG_MODULE = "ultra.baselines.ppo.ppo.tune_params"
```

Comment:
Do we want to create a separate config in the test directory, instead of using the baseline one? I know it will not change anything drastically because the result params will be stored in a different file, but it will ensure that the tests are a standalone thing

Reply:
Oh yes, great idea!